doc/api/fs.md

```diff
@@ -151,6 +151,10 @@ Stop watching for changes on the given `fs.FSWatcher`.
 
 Emitted when the ReadStream's file is opened.
 
+### Event: 'close'
+
+Emitted when the ReadStream's underlying file descriptor has been closed. Comes from fs.close() method. 
```

Comment:
Is the note about `fs.close()` really that helpful/useful? Maybe we could just drop that part entirely?